src/masternodes/mn_rpc.cpp

```diff
@@ -3081,39 +3081,288 @@ UniValue isappliedcustomtx(const JSONRPCRequest& request) {
+
+    RPCHelpMan{"sendtokenstoaddress",
+               "\nCreates (and submits to local node and network) a transfer transaction from your accounts balances (may be picked manualy or autoselected) to the specfied accounts.\n" +
+               HelpRequiringPassphrase(pwallet) + "\n",
+               {
+                    {"from", RPCArg::Type::OBJ, RPCArg::Optional::NO, "",
+                        {
+                            {"address", RPCArg::Type::STR, RPCArg::Optional::OMITTED, "The source defi address is the key, the value is amount in amount@token format. "
+                                                                                     "If obj is empty (no address keys exists) - trying to autoselecting accounts from wallet "
+                                                                                     "with necessary balances to transfer."},
+                        },
+                    },
+                    {"to", RPCArg::Type::OBJ, RPCArg::Optional::NO, "",
+                        {
+                            {"address", RPCArg::Type::STR, RPCArg::Optional::NO, "The defi address is the key, the value is amount in amount@token format. "
+                                                                                     "If multiple tokens are to be transferred, specify an array [\"amount1@t1\", \"amount2@t2\"]"},
+                        },
+                    },
+                    {"selectionMode", RPCArg::Type::STR, /* default */ "pie", "If param \"from\" is empty this param indicates accaouts autoselection mode."
+                                                                              "May be once of:\n"
+                                                                              "  \"forward\" - Selecting accounts without sorting, just as address list sorted.\n"
+                                                                              "  \"crumbs\" - Selecting accounts by ascending of sum token amounts.\n"
+                                                                              "    It means that we will select first accounts with minimal sum of neccessary token amounts.\n"
+                                                                              "  \"pie\" - Selecting accounts by descending of sum token amounts.\n"
+                                                                              "    It means that we will select first accounts with maximal sum of neccessary token amounts."
+                    },
+                },
+                RPCResult{
+                       "\"hash\"                  (string) The hex-encoded hash of broadcasted transaction\n"
+                },
+                RPCExamples{
+                        HelpExampleCli("sendtokenstoaddress", "\"{}\" "
+                                                    "\"{\\\"dstAddress1\\\":\\\"1.0@DFI\\\",\\\"dstAddress2\\\":[\\\"2.0@BTC\\\", \\\"3.0@ETH\\\"]}\" "
```

Comment:
Can you format helps like that
```
sendtokenstoaddress", "'{}' '{\"dstAddress1\":\"1.0@DFI\",\"dstAddress2\":[\"2.0@BTC\", \"3.0@ETH\"]}' crumbs'
```
escape parameters by `' '` then escaping is minimal.

Reply:
Done